addons-l10n/en/editor-searchable-dropdowns.json

```diff
@@ -0,0 +1,7 @@
+{
+  "editor-searchable-dropdowns/createGlobalVariable": "Create variable \"{name}\" for all sprites",
+  "editor-searchable-dropdowns/createLocalVariable": "Create variable \"{name}\" for this sprite only",
+  "editor-searchable-dropdowns/createGlobalList": "Create list \"{name}\" for all sprites",
+  "editor-searchable-dropdowns/createLocalList": "Create list \"{name}\" for this sprite only",
+  "editor-searchable-dropdowns/createBroadcast": "Create broadcast \"{name}\""
```

Comment:
The closest string in Scratch is "New message", so perhaps "Create message"? Not sure.